content/tutorials/presence.textile

```diff
@@ -0,0 +1,438 @@
+---
+title: Presence Tutorial
+section: tutorials
+index: 10 # TODO ??
```

Comment:
Nope, it just affects the position in the nav, but the tutorials are not listed in the nav anyway so you can ignore it